deploy/dev/reload.sh

```diff
@@ -3,32 +3,49 @@ set -xeuo pipefail
 
 . utils.sh
 
-# Script for making it easy to make a change locally and redeploy
-pushd ..
-  ./bin/build
-popd
+main() {
+  # Clean-up previous run
+  helm delete $(helm ls -aq)
+  $cli_with_timeout "delete deployment test-env --ignore-not-found=true"
 
-set_namespace $APP_NAMESPACE_NAME
+  pushd ..
+    ./bin/build
+  popd
 
-docker tag "secrets-provider-for-k8s:dev" "${APP_NAMESPACE_NAME}/secrets-provider"
+  set_namespace $APP_NAMESPACE_NAME
 
-selector="role=follower"
-cert_location="/opt/conjur/etc/ssl/conjur.pem"
-if [ "$CONJUR_DEPLOYMENT" = "oss" ]; then
-  selector="app=conjur-cli"
-  cert_location="/root/conjur-${CONJUR_ACCOUNT}.pem"
-fi
+  docker tag "secrets-provider-for-k8s:dev" "${APP_NAMESPACE_NAME}/secrets-provider"
 
-conjur_pod_name=$($cli_with_timeout get pods --selector=$selector --namespace $CONJUR_NAMESPACE_NAME --no-headers | awk '{ print $1 }' | head -1)
-ssl_cert=$($cli_with_timeout "exec ${conjur_pod_name} --namespace $CONJUR_NAMESPACE_NAME cat $cert_location")
+  if [ "${DEV_HELM}" = "true" ]; then
```

Comment:
you sometimes check if this is true and sometimes if it is false. please align.

Reply:
For readability purposes I put the shorter checks in the first if block but I can align

Reply:
i think it's easier if it's aligned